### ISSUE TITLE
List of table improvements

### ISSUE THREAD

**Issue by wouter1975** [workfloworchestrator/orchestrator-ui-library#1253]
This ticket lists idea what to improve to the table component

- [x] mouse-over on each cell will show the full text of the cell --> in progress
- [ ] resize column width by user
  - [ ] store the column width preference per table in local storage
  - [ ] Truncate header name to meet the width of the column, eg `Subscrip.. v` for Subscription column
  - [ ] mouse-over on column header will show the full name of the table header - looks already implemented
- [x] Add sort/filter icon to table header columns https://github.com/workfloworchestrator/orchestrator-ui-library/issues/458 to be discussed --> on hold, wait for user feedback after #459
- [x] Always show the sorted column indicator (thus https://github.com/workfloworchestrator/orchestrator-ui-library/issues/315) possibly already fixed --> in progress
- [x] Single click sorting, see https://github.com/workfloworchestrator/orchestrator-ui-library/issues/459 --> in progress

- [x] Define a minimal table width, add horizontal scrollbar if applicable, see https://github.com/workfloworchestrator/orchestrator-ui-library/issues/509

- [ ] ~wrap text in cells, with a toggle function~
